raster-foundry/raster-foundry
# Add ProjectLayer datamodel, dao, migration

## Overview
Add a ProjectLayer datamodel
Add ProjectLayerDao and migration

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~


## Testing Instructions

 * Verify that it compiles and can be used to serve in the spec below
* Verify that the migration runs and creates the table correctly

Related to https://github.com/raster-foundry/raster-foundry-api-spec/pull/74
Closes #4450
Closes #4451
